dbtmetabase/metabase.py

```diff
@@ -221,13 +221,21 @@ def export_model(
         else:
             model_description = model.description
 
+        body_table = {}
+        if api_table["description"] != model_description:
+            body_table["description"] = model_description
+        if api_table["points_of_interest"] != model.points_of_interest:
```

Comment:
Do these get returned if they haven't been set for all versions of Metabase? Just making sure it doesn't blow up for anyone using older Metabase versions. If not, then checking for `"points_of_interest" in api_table` first might be useful.

Reply:
Agree, better to lean safe here with a `get` accessor.

Reply:
Nice! I've adjusted to use `get()` 👍